compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/ProgramFileConstants.java

```diff
@@ -25,9 +25,9 @@
 
     public static final int MAGIC_NUMBER = 0xBA1DA4CE;
     public static final short VERSION_NUMBER = 50;
-    public static final int BIR_VERSION_NUMBER = 52;
-    public static final short MIN_SUPPORTED_VERSION = 52;
-    public static final short MAX_SUPPORTED_VERSION = 52;
+    public static final int BIR_VERSION_NUMBER = 51;
+    public static final short MIN_SUPPORTED_VERSION = 51;
+    public static final short MAX_SUPPORTED_VERSION = 51;
```

Comment:
Do we need this change?

Reply:
This is to make sure the BIR version of the reader and the writer. This was changed earlier, as the stream type was removed. However, since this is sort of a revert PR, and we re-introduce stream type, earlier BIR version should still support, without any issues